src/apa102/CMakeLists.txt

```diff
@@ -5,4 +5,4 @@ upm_mixed_module_init (NAME apa102
     CPP_HDR apa102.hpp
     CPP_SRC apa102.cxx
     FTI_SRC apa102_fti.c
-    REQUIRES upmc-utilities mraa)
+    REQUIRES utilities-c mraa)
```

Comment:
Since we automatically add upmc-utilities as a dependency on everything, utilities-c does not need to be specified here, correct?